tests/shell/shell_recorder/script.dat

```diff
@@ -0,0 +1,13 @@
+Storage: dump
+Mountpoint: system/testmount/test
+File: /tmp/test.dump
+MountArgs:
+DiffType: File
+< mount $File $Mountpoint $Storage
```

Comment:
yeah, i was already thinking about splitting the header and the commands, i'm not completely set on the syntax yet.